salt/cloud/clouds/clc.py

```diff
@@ -0,0 +1,368 @@
+# -*- coding: utf-8 -*-
+'''
+CenturyLink Cloud Module
+===================
+
+.. versionadded:: 2017.7.14
+
+The CLC cloud module allows you to manage CLC Via the CLC SDK.
+
+:codeauthor: Stephan Looney <slooney@stephanlooney.com>
+
+
+Dependencies
+============
+
+- clc-sdk Python Module
+
+CLC SDK
+-------
+
+clc-sdk can be installed via pip:
+
+.. code-block:: bash
+
+    pip install clc-sdk
+
+.. note::
+  For sdk reference see: https://github.com/CenturyLinkCloud/clc-python-sdk
+
+Configuration
+=============
+
+To use this module, set up the clc-sdk, username,password,key in the
```

Comment:
spaces between commas? also, this references `user` and this says `username`. Should they be consistent?